.github/workflows/mini-kit.yml

```diff
@@ -0,0 +1,20 @@
+name: MiniKit Doctor
```

Comment:
suggestion: Could we scale this PR back to just the docs update for now? There's a lot going on in this action and I don't think it's currently running, so it's also hard to verify.

I'd also ask that we get rid of the `starters/*` folder, since we have templates and examples in the onchainkit repo. We've spread that sort of thing around in the past, and IMO it'd be nice to keep it it one place. The onchainkit repo makes the most sense to me, as we run tests against the templates when we make changes to OnchainKit.